gson/src/main/java/com/google/gson/annotations/SerializedName.java

```diff
@@ -64,7 +64,7 @@
  * @author Joel Leitch
  */
 @Retention(RetentionPolicy.RUNTIME)
-@Target(ElementType.FIELD)
+@Target({ElementType.FIELD, ElementType.METHOD})
```

Comment:
@swankjesse Do we really want to do this?

If we allow SerializedName on methods, we should actually support it in full. 
We Don't and we seem to have no plans to support it either. So, why open it up?